tests/registry.test.ts

```diff
@@ -0,0 +1,333 @@
+import { assert, expect } from "@esm-bundle/chai";
+import { init, initializeLogger, Wasmer } from "..";
+
+const initialized = (async () => {
+	await init(new URL("../dist/wasmer_js_bg.wasm", import.meta.url), undefined,
+		{ registry_url: "https://registry.wasmer.wtf/graphql", token: "<YOUR_TOKEN>" }
```

Comment:
Is this passing. The token is invalid

Reply:
I think it'd be better not embedding a token in a public repo, even for a "fake" CI-only user.

Reply:
Agreed, but to test we can get the token from an environment variable and use it, which we are not doing here

Reply:
done